packages/wdio-mocha-framework/src/index.ts

```diff
@@ -202,8 +203,11 @@ class MochaAdapter {
 
         switch (hookName) {
         case 'beforeSuite':
+            this._suiteStartDate = new Date()
```

Comment:
Let's directly work with the timestamp here.
```suggestion
            this._suiteStartDate = Date.now()
```